pyfr/nputil.py

```diff
@@ -62,7 +62,8 @@ def newfn(*args, **kwargs):
     'tan': np.tan, 'atan': np.arctan, 'atan2': np.arctan2,
     'abs': np.abs, 'pow': np.power, 'sqrt': np.sqrt,
     'tanh': np.tanh, 'pi': np.pi,
-    'max': np.maximum, 'min': np.minimum
+    'max': np.maximum, 'min': np.minimum,
+    'mod': np.mod
```

Comment:
Is this needed?  Mod, like if, can be a nuisance function.

Reply:
yeah, I'm not sure why this slid in